atd-vzd/Dockerfile

```diff
@@ -1,5 +1,3 @@
-#FROM frankinaustin/postgis:14-3.3
-#FROM postgis/postgis:14-3.3
```

Comment:
We can keep this around if it is helpful to have a history of images here. I swept through the files this folder since I missed the readme in the migrations PR.

Reply:
Thanks for sweeping out this cruft! 🧹